### DISCUSSION TITLE
Autorun on boot does not give me normal IP addresses

### DISCUSSION THREAD

**Discussion by bobodapopper** [vicwomg/pikaraoke#203] [General]
Hi, I'm not sure what I'm doing wrong but when I added the line in /etc/rc.local to have this PiKaraoke run on book instead of getting the usual 192.168.x.x:5000 at the splash screen I'd get feasdda:daxdsf:dfasdf:kidasdf:5000 for example.  Can anyone let me know what's going on here?  Thanks!

FYI if I go to, in my case, http://192.168.1.63:5000 I'll still able to use my phone to queue up songs and everything but the QR code and IP address on the splash screen are all wrong.

**Comment by vicwomg** [comment]
That looks like an IPv6 address. Not sure why that would happen on boot and not otherwise. You could try adding "sleep 20" before the line to delay launching pikaraoke for 20s and perhaps have the pi's IP address "catch up" in the booting process.

If that doesn't work, and you have an extra SD card, I'd be curious if it occurs with the pre-made 1.1.2 image, which already has auto-boot enabled. If so, then it must be some general router network configuration issue.
